code/modules/lighting/lighting_atom.dm

```diff
@@ -46,21 +46,13 @@
 		T.reconsider_lights()
 	return ..()
 
-/atom/movable/Move()
-	var/turf/old_loc = loc
+/atom/Entered(atom/movable/obj, atom/prev_loc)
```

Comment:
atom/Exited will be called. There's no point calling atom/Entered on null, nothing can be done.